libs/sdk/testing/src/lib/a11y/a11y-analyzer.ts

```diff
@@ -29,6 +46,22 @@ function parseMessage(violations: axe.Result[]): string {
   return message;
 }
 
+function filterViolationNodeResults(
+  result: axe.Result
+): (node: axe.NodeResult) => boolean {
+  if (
+    [
+      'aria-required-children', // AG Grid uses some aria-hidden elements that axe doesn't like
```

Comment:
thanks for adding comments here so we're not trying to remember why we removed things 6 months from now 😂